src/cargo/util/context/schema.rs

```diff
@@ -224,6 +224,8 @@ pub struct CargoBuildConfig {
 #[serde(rename_all = "kebab-case")]
 pub struct CargoBuildAnalysis {
     pub enabled: Option<bool>,
+    pub console: Option<bool>,
+    pub file: Option<bool>,
```

Comment:
The PR describes what this does but doesn't provide the motivation.

I find this
- confusing: it isn't very clear what these states mean and why you would use them
- odd to have build analysis and to be able to turn off `file`
- unclear on what purpose this is meant to serve, especially at this point

Reply:
> I'd rather just wait, see how stabilization goes on this, and evaluate always putting these messages out.

That is fair. We should wait and see.